core/src/sigverify_stage.rs

```diff
@@ -74,24 +74,43 @@ pub struct DisabledSigVerifier {}
 
 #[derive(Default)]
 struct SigVerifierStats {
-    recv_batches_us_hist: histogram::Histogram, // time to call recv_batch
-    verify_batches_pp_us_hist: histogram::Histogram, // per-packet time to call verify_batch
-    discard_packets_pp_us_hist: histogram::Histogram, // per-packet time to call verify_batch
-    dedup_packets_pp_us_hist: histogram::Histogram, // per-packet time to call verify_batch
-    batches_hist: histogram::Histogram,         // number of packet batches per verify call
-    packets_hist: histogram::Histogram,         // number of packets per verify call
+    /// The time (us) spent calling recv_batch() to get PacketBatches into the stage
```

Comment:
I'm always a little leary of calling out specific function/variable names in comments (because they're easy to forget to update if we change naming and/or behavior).

It might also be nice for the histogram metrics to indicate what each data point captures (e.g. one invocation of verifying currently received packet batches). Maybe we can break it out into sections like:

1. Metrics relating to the amount of work done during each invocation
2. Metrics aggregating across all invocations since the last report/reset

That way we won't have to get too repetitive in the comment for each individual metrics while still making it clear what it captures.

Reply:
> I'm always a little leary of calling out specific function/variable names in comments (because they're easy to forget to update if we change naming and/or behavior).

Typically same, the name of the function is in the variable name which is whey I went that way. I'll strip out the function name tho

> ...
> That way we won't have to get too repetitive in the comment for each individual metrics while still making it clear what it captures.

Good idea, I'll do that